### ISSUE TITLE
Broken layout in Org Agenda (C-a a)

### ISSUE THREAD

**Issue by felipelalli** [iqbalansari/emacs-emojify#53]
The problem here is that, e.g., `:thisemoji:` emacs understand as 11 char length and not 1 as it should understand. I guess other tools that relies on char width become broken with `emojify-mode` as well. I'm not sure if it is possible to fix it, but...

![a1](https://user-images.githubusercontent.com/207639/59143705-ee2b4b00-89a3-11e9-837a-55558fb0e010.png)

**Comment by iqbalansari** [comment]
Yes I am aware of this issue, unfortunately there does not seem to be a way to fix

**Comment by felipelalli** [comment]
@iqbalansari If you replace the text :smile: by the Unicode char equivalent like 😄 isn't enough to fix the issue? Or not every emoji has an equivalent? There is a command to replace the full-name to the Unicode character? Thank you.

**Comment by iqbalansari** [comment]
Actually replacing the text with the unicode equivalent would require changing the "actual" contents of the buffer, I do not think that is a good idea

**Comment by felipelalli** [comment]
@iqbalansari What if a command like `M-x emojify-replace-text-to-unicode` that acts above the current word in cursor and/or region. It would be a nice feature. Thank you!

**Comment by iqbalansari** [comment]
That is good idea, I will add a command to do that
